### PR TITLE
test: set etcdutl to default

### PR DESCRIPTION
**Pull request by kkkkun**
Parts of https://github.com/etcd-io/etcd/issues/13798
Fixes comment: https://github.com/etcd-io/etcd/pull/13809#discussion_r827977075

### CODE REVIEW

File: tests/e2e/ctl_v3_snapshot_test.go [hunk 1]
```diff
@@ -29,8 +29,7 @@ import (
 	"go.etcd.io/etcd/tests/v3/framework/e2e"
 )
 
-func TestCtlV3Snapshot(t *testing.T)        { testCtl(t, snapshotTest) }
```
Comment (serathius) [inline tests/e2e/ctl_v3_snapshot_test.go:32]: I don't think we should remove a test in this PR. Why remove this one?

**Review by serathius — APPROVED** [review]
cc @ptabor @spzala @ahrtr

File: tests/e2e/ctl_v3_snapshot_test.go (hunk 1 above)
Comment (kkkkun) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: > I don't think we should remove a test in this PR. Why remove this one?

The test `TestCtlV3Snapshot` is same as `TestCtlV3SnapshotEtcdutl` when delete  `withEtcdutl`

Comment (kkkkun) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: The test `TestCtlV3Snapshot` is same as `TestCtlV3SnapshotEtcdutl` when delete `withEtcdutl`.

**Review by ahrtr — APPROVED** [review]
LGTM

File: tests/e2e/ctl_v3_snapshot_test.go (hunk 1 above)
Comment (serathius) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: Is `TestCtlV3Snapshot` is currently using etcutl?

Comment (kkkkun) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: It‘s used both. 
Command `snapshot save` would use etcdctl like https://github.com/kkkkun/etcd/blob/62641d33854e206f915398332dbc2b976e9ae5c8/tests/e2e/ctl_v3_snapshot_test.go#L131,

and `snapshot restore/status` would use etcdutl. https://github.com/kkkkun/etcd/blob/62641d33854e206f915398332dbc2b976e9ae5c8/tests/e2e/ctl_v3_snapshot_test.go#L136

Comment (serathius) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: So they don't test the same thing. One uses `etcdctl` other `etcutl`, based on that I don't think we want to remove the test.

Comment (kkkkun) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: They test the same thing.  Both `TestCtlV3Snapshot` and `TestCtlV3SnapshotEtcdutl` call `snapshotTest`.

In `snapshotTest`, it firstly do `etcdctl snapshot save` used etcdctl , and then  `etcdutl snapshot status` used etcdutl.

Comment (serathius) [reply tests/e2e/ctl_v3_snapshot_test.go:32]: Ack, thanks for confirming.

**Review by spzala — APPROVED** [review]
Thanks @kkkkun @serathius 
lgtm

**Pull request merged**